### PR TITLE
Updated READMEs on releasing, maintaining, admins and security.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@amazon.com>

### Description

Beginning of standardizing processes across the org for branching, admins, maintainers. Added security.
 
### Issues Resolved

Partly #95.
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,38 @@
...
->
->**4. Used everywhere**. Our goal is for as many people as possible to use it in their business, their software, and their projects. Use it however you want. Surprise us!
->
->**5. Made with your input**. We will ask for public input on direction, requirements, and implementation for any feature we build.
->
->**6. Open to contributions**. Great open source software is built together, with a diverse community of contributors. If you want to get involved at any level - big, small, or huge - we will find a way to make that happen. We don’t know what that looks like yet, and we look forward to figuring it out together.
->
->**7. Respectful, approachable, and friendly**. This will be a community where you will be heard, accepted, and valued, whether you are a new or experienced user or contributor.
->
->**8. A place to invent**. You will be able to innovate rapidly. This project will have a stable and predictable foundation that is modular, making it easy to extend.
```
Comment (dblock) [inline README.md:49]: These are [Project Principles](https://opensearch.org/#principles) on the website.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,38 @@
...
+- [Welcome!](#welcome)
+- [Project Resources](#project-resources)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+- [Copyright](#copyright)
+
+## Welcome!
+
+**OpenSearch** is [a community-driven, open source fork](https://aws.amazon.com/blogs/opensource/introducing-opensearch/) of [Elasticsearch](https://en.wikipedia.org/wiki/Elasticsearch) and [Kibana](https://en.wikipedia.org/wiki/Kibana) ensuing the
```
Comment (camerski) [inline README.md:11]: Suggestion: "ensuing" -> "following".

Ensuing also means "to strive to attain", or "pursue". Using "following" makes this sentence clearer.

Comment (dblock) [reply README.md]: Changed.

**Comment by dblock** [comment]
@CEHENKLE could I get a review for this please?

**Review by camerski — APPROVED** [review]

**Comment by dblock** [comment]
start gradle check

**Pull request merged**